governance/SDK.md

```diff
@@ -10,7 +10,42 @@ The community is organized as follows:
 
 ## New maintainers
 
-TBD
+Maintainers are a crucial part of this community: they keep alive our projects
+and ensure both development and maintenance. We define a set of criteria,
+followed by a vote, to include an active contributor as maintainer of one or
+more sdk projects. The active contributor may propose himself to become a
+maintainer, although we encourage the community to include him as part of the
+group.
+
+The criteria for a contributor to `sdk-x` to become part of `sdk-x-maintainers`
+is:
+
+1. At least 20 reviews on eligible PRs
+1. At least 20 submitted eligible PRs as author
```

Comment:
This is a pretty high bar for some repositories. Even as active as sdk-javascript is, there have only been 20 PRs in the last month, and it would be unusual for a community member to contribute anywhere near that much. I might suggest something looser such as.

> 1. Regular reviews of eligible pull requests over a period of 30 days
> 1. Regular contributions to the project in the form of eligible pull requests over a period of 30 days

Reply:
I disagree for a couple of reasons:

* I think the criteria should be well defined, number based. This makes sure rules are always followed and there are no shady situations. I wanna make sure nobody can say "this guy had a different treatment from other people"
* 20 both for PRs and reviews seems pretty reasonable. We're not defining how you become a contributor, but how you become a maintainer and you get full powers in the project like everybody else. If the project is active, 20 PRs and 20 reviews is something you get in ~2 months, which IMO is a good time frame to go from 0 to maintainer.

Reply:
My experience with the Node.js project was quite different than this. I consider that project to be very well managed, and their requirements for becoming a maintainer are definitely looser than this, while at the same time consideration is given to a lot of activities outside of actual PRs.

https://nodejs.org/en/get-involved/contribute/#becoming-a-collaborator

* code commits and pull requests
* documentation commits and pull requests
* comments on issues and pull requests
* contributions to the Node.js website
* assistance provided to end users and novice contributors
* participation in Working Groups
* other participation in the wider Node.js community

And they allow for contributors to bring up themselves for consideration if they feel the requirements have been met.

> If individuals making valuable contributions do not believe they have been considered for commit access, they may log an issue or contact a TSC member directly.